<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_go">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_exit">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>280</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_output_directory">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>311</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>C:/SharedData/Scientific_Work/Biology/C19/computer_simulations/PNM/data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_select_output_directory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>output directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>outputfile name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_outputfile_name">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>initial transmission factor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>modified transmission factor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>extended transmission factor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pre_activation period</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>active period</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_p_trans_a_s0">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_p_trans_b_s0">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_p_trans_x_s0">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_ti_s0">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_ta_s0">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pre_activation period</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_p_trans_a_s1">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_p_trans_b_s1">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>initial transmission factor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>modified transmission factor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_ti_s1">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>active period</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_ta_s1">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>110</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_xchng_final_a">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>190</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>exchange | modified | initial state</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_eternity">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_xchng_initial_a">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>eternity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>exchange | initial | initial state</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>exchange | initial | final state</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_xchng_initial_b">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>190</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>exchange | modified | final state</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_xchng_final_b">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>transition to modified constraints</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_transition_start">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_transition_period">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>transition length</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>    evironment </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string> strain 0 </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string> strain 1 </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_num_contacts_a">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>190</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>contact events | initial</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_num_contacts_b">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>190</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>contact events | final</string>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>bt_go</zorder>
   <zorder>bt_exit</zorder>
   <zorder>ed_output_directory</zorder>
   <zorder>bt_select_output_directory</zorder>
   <zorder>label</zorder>
   <zorder>ed_outputfile_name</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>ed_p_trans_a_s0</zorder>
   <zorder>ed_p_trans_b_s0</zorder>
   <zorder>ed_p_trans_x_s0</zorder>
   <zorder>ed_ti_s0</zorder>
   <zorder>ed_ta_s0</zorder>
   <zorder>label_7</zorder>
   <zorder>ed_p_trans_a_s1</zorder>
   <zorder>ed_p_trans_b_s1</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>ed_ti_s1</zorder>
   <zorder>label_11</zorder>
   <zorder>ed_ta_s1</zorder>
   <zorder>ed_xchng_final_a</zorder>
   <zorder>label_10</zorder>
   <zorder>ed_eternity</zorder>
   <zorder>ed_xchng_initial_a</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>label_14</zorder>
   <zorder>ed_xchng_initial_b</zorder>
   <zorder>label_15</zorder>
   <zorder>ed_xchng_final_b</zorder>
   <zorder>label_16</zorder>
   <zorder>ed_transition_start</zorder>
   <zorder>ed_transition_period</zorder>
   <zorder>label_17</zorder>
   <zorder>ed_num_contacts_a</zorder>
   <zorder>label_18</zorder>
   <zorder>ed_num_contacts_b</zorder>
   <zorder>label_19</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>ed_eternity</tabstop>
  <tabstop>ed_transition_start</tabstop>
  <tabstop>ed_transition_period</tabstop>
  <tabstop>ed_xchng_initial_a</tabstop>
  <tabstop>ed_xchng_final_a</tabstop>
  <tabstop>ed_num_contacts_a</tabstop>
  <tabstop>ed_xchng_initial_b</tabstop>
  <tabstop>ed_xchng_final_b</tabstop>
  <tabstop>ed_num_contacts_b</tabstop>
  <tabstop>ed_p_trans_a_s0</tabstop>
  <tabstop>ed_p_trans_b_s0</tabstop>
  <tabstop>ed_p_trans_x_s0</tabstop>
  <tabstop>ed_ti_s0</tabstop>
  <tabstop>ed_ta_s0</tabstop>
  <tabstop>ed_p_trans_a_s1</tabstop>
  <tabstop>ed_p_trans_b_s1</tabstop>
  <tabstop>ed_ti_s1</tabstop>
  <tabstop>ed_ta_s1</tabstop>
  <tabstop>ed_output_directory</tabstop>
  <tabstop>ed_outputfile_name</tabstop>
  <tabstop>bt_select_output_directory</tabstop>
  <tabstop>bt_go</tabstop>
  <tabstop>bt_exit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
